src/javis_viewer.jl

```diff
@@ -0,0 +1,146 @@
+# TODO: Intelligent way of handling slider
+
+#=
+FIXME:
+Fascinating bug I found.
+Apparently to create a list of actions anywhere, you must first define a Video object.
+Else, you get an error like this:
+
+```
+ERROR: LoadError: BoundsError: attempt to access 0-element Array{Video,1} at index [1]
```

Comment:
This is expected as an Action needs a Video. The error message should be improved though.

Reply:
Opened a bug report on this guy #152 Feel free to close this convo.